src/pages/workspaces/WorkspacesList/CatagorizedWorkspaces.ts

```diff
@@ -0,0 +1,26 @@
+import _ from 'lodash/fp';
+import { canWrite, WorkspaceWrapper as Workspace } from 'src/libs/workspace-utils';
+
+export interface CatagorizedWorkspaces {
```

Comment:
"Categorize" is misspelled here and in the function name.

Reply:
You found my greatest weakness!